packages/mobile/src/components/now-playing-drawer/NowPlayingDrawer.tsx

```diff
@@ -219,10 +236,16 @@ const NowPlayingDrawer = ({
               <TrackInfo track={track} user={user} />
             </View>
             <View style={styles.scrubberContainer}>
-              <Scrubber mediaKey='1' />
+              <Scrubber
+                mediaKey={`${mediaKey}`}
+                isPlaying={isPlaying}
+                onPressIn={() => setIsGestureEnabled(false)}
```

Comment:
if we make these callbacks, i think scrubber then doesn't need to be memoized?

Reply:
Yep you're right. Only was memoizing b/c the Slider used to be the same component. Ty